recipes/robotlab/builder.py

```diff
@@ -19,6 +21,13 @@
 }
 SVG = SRC_DIR / "robotlab" / "src" / "robotlab" / "icons" / "starter.svg"
 
+OPENCV_OSX = """
```

Comment:
Not prod of this: any thoughts of ways to make it cleaner? Can we add something at the RFSL layer?